modules/serverless-create-channel-kn.adoc

```diff
@@ -6,12 +6,12 @@
 [id="serverless-create-channel-kn_{context}"]
 = Creating a channel by using the Knative CLI
 
-You can create a channel with the cluster default configuration by using the `kn` CLI.
+Using the `kn` CLI to create channels provides a more streamlined and intuitive user interface over modifying YAML files directly. You can use the `kn channel create` command to create a channel by using the `kn` CLI.
```

Comment:
I didn't find a style rule, but I wonder if s/over/than would read better.

Reply:
I think you're right actually. I have been using a sort of boilerplate doc for these types of modules since there are SO many that need updating, but I will use this going forward to improve it. Thanks!